src/inputs/ToggleChipGroup.tsx

```diff
@@ -48,33 +53,41 @@ interface ToggleChipProps {
   value: string;
   groupState: CheckboxGroupState;
   selected: boolean;
+  isDisabled?: boolean;
+  disabledReason?: string;
 }
 
 function ToggleChip(props: ToggleChipProps) {
-  const { label, value, groupState, selected: isSelected, ...others } = props;
+  const { label, value, groupState, selected: isSelected, isDisabled = false, disabledReason, ...others } = props;
   const ref = useRef(null);
-  const { inputProps } = useCheckboxGroupItem({ value, "aria-label": label }, groupState, ref);
+  const { inputProps } = useCheckboxGroupItem({ value, "aria-label": label, isDisabled }, groupState, ref);
   const { isFocusVisible, focusProps } = useFocusRing();
 
-  return (
-    <label
-      css={{
-        ...Css.relative.dib.br16.sm.px1.cursorPointer.pyPx(4).bgGray200.$,
-        ...(isSelected
-          ? {
-              ...Css.white.bgLightBlue700.$,
-              ":hover": Css.bgLightBlue800.$,
-            }
-          : { ":hover": Css.bgGray300.$ }),
-        ...(isFocusVisible ? Css.bshFocus.$ : {}),
-      }}
-      data-selected={isSelected}
-      {...others}
-    >
-      <VisuallyHidden>
-        <input {...inputProps} {...focusProps} />
-      </VisuallyHidden>
-      {label}
-    </label>
-  );
+  return maybeTooltip({
+    title: disabledReason,
+    placement: "top",
+    children: (
+      <label
+        css={{
+          ...Css.relative.dib.br16.sm.px1.cursorPointer.pyPx(4).bgGray200.if(isDisabled).cursorNotAllowed.gray600.pr1.$,
+          ...(isSelected
+            ? {
+                ...Css.white.bgLightBlue700.$,
+                ":hover:not(:isDisabled)": Css.bgLightBlue800.$,
+              }
+            : { ":hover:not(:isDisabled)": Css.bgGray300.$ }),
```

Comment:
Question: is `:isDisabled` correct? I've seen `:disabled`:

https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled

But not `:isDisabled`. Does this somehow magically work with the `data-disabled` attribute? 

I was expecting per:

https://stackoverflow.com/a/17249001

Something like `:not([data-disabled="true"])`

Reply:
I inserted `:isDisabled` because react-aria's `useCheckboxGroupItem` props extends InputBase  type down the chain which does have the `isDisabled` prop. It'll actually work without using `data-disabled` attribute so maybe it's just a quirk of react-aria?

But yea probably better to do it the traditional CSS way.